main.js

```diff
@@ -5152,9 +5152,17 @@ function addProfile(profession, profile, base){
                 limit: 0
             };
             _vendorItems[_vendorItems.length] = {
-                pattern: /_Green_T[1-5]_Unid$/,
+                pattern: /_Green_T[1-7]_Unid$/,

```

Comment:
I don't think anyone that uses the AD income from leadership with the script will try to store unstackable green stuff, that gives less than a single peridot.
But it always can be split into a different option.
1-6 + 7